src/coreclr/nativeaot/Runtime/CommonMacros.h

```diff
@@ -143,11 +143,7 @@ inline bool IS_ALIGNED(T* val, uintptr_t alignment);
 
 #define DATA_ALIGNMENT  8
```

Comment:
By the way, `DATA_ALIGNMENT` looks like dead code to me.

Reply:
Yes, it is dead code. Deleting in https://github.com/dotnet/runtime/pull/101655